barretenberg/cpp/src/barretenberg/commitment_schemes/zeromorph/zeromorph.hpp

```diff
@@ -462,10 +465,19 @@ template <typename Curve> class ZeroMorphVerifier_ {
     static Commitment compute_C_zeta_x(const Commitment& C_q,
                                        std::vector<Commitment>& C_q_k,
                                        FF y_challenge,
-                                       FF x_challenge)
+                                       FF x_challenge,
+                                       const FF log_circuit_size,
+                                       const FF circuit_size)
     {
-        size_t log_N = C_q_k.size();
-        size_t N = 1 << log_N;
+        size_t N{ 0 };
+        size_t log_N{ 0 };
+        if constexpr (Curve::is_stdlib_type) {
```

Comment:
need a TODO here because this is very suspicious/wrong

Reply:
added a TODO on the pow below which is the real issue